.github/workflows/reusable_docker_package.yaml

```diff
@@ -0,0 +1,151 @@
+name: Reusable docker package make and upload
+
+on:
+  workflow_call:
+    inputs:
+      os:
+        type: string
+        description: "Target os. Default value is debian-11."
```

Comment:
What's the reason that debian 11 is the default?

Is it even possible to make debian 12 the default?

Reply:
It is and I will :+1: